packages/client/components/StartMeetingFAB.tsx

```diff
@@ -75,8 +75,8 @@ const StartMeetingFAB = (props: Props) => {
   return (
     <Block>
       <Button onClick={onClick} onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
-        <MeetingIcon isExpanded={isBig}>{'forum'}</MeetingIcon>
-        <MeetingLabel isExpanded={isBig}>{'Start New Meeting'}</MeetingLabel>
```

Comment:
I like removing the word `Start`
i think we need to reduce the width of the button though, because currently it looks like there's just a lot of empty space on the right